packages/extension-ui/src/components/PasswordField/TransitionMessage.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019-2023 @polkadot/extension-ui authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useEffect, useRef, useState } from 'react';
+import { Transition } from 'react-transition-group';
+import styled from 'styled-components';
+
+import Message, { MessageType } from './Message';
+
+type Props = {
+  className?: string;
+  in: boolean;
+  messageType: MessageType;
+  text: string;
+  duration: number;
+};
+
+const TransitionMessage = ({className = '', duration, in: show, messageType, text}: Props) => {
+  const nodeRef = useRef(null);
+  const [currentText, setCurrentText] = useState<string>();
+
+  useEffect(() => {
+    if (text === currentText) {
+      return;
+    }
+
+    const timeout = currentText ? 150 : 0;
+
+    setTimeout(setCurrentText, timeout, text);
+  }, [setCurrentText, currentText, text]);
+
+  const defaultStyle = {
+    transition: `grid-template-rows ${duration}ms, opacity ${duration}ms, padding ${duration}ms`,
```

Comment:
Oh wow, wasn't aware that `grid-template-rows` are translatable, especially with the `fr` unit :O Awesome. TIL.

Reply:
I learned two days ago :laughing: 
I highly recommend https://keithjgrant.com/posts/2023/04/transitioning-to-height-auto/